distribution/openrct2.d.ts

```diff
@@ -3100,14 +3100,14 @@ declare global {
         staffType: StaffType;
 
         /**
-         * Colour of the staff member. Not applicable for entertainers.
+         * Colour of the staff member. Not applicable to entertainers.
          */
         colour: number;
 
         /**
-         * The entertainer's costume, only applicable for entertainers.
+         * The staff member's costume.
          */
-        costume: number;
+        costume: number | string;
```

Comment:
Maybe nice to have a string union here with all possible options instead of a blank string?

Reply:
That would work right now, but remember introducing custom staff objects is the goal. There would not be a definite list.

(Assigning an empty string will fail, btw)

Reply:
We could do this:
```suggestion
        costume: StaffCostume | string | number;
```
This would allow custom costumes as well as built-in intellisense when it comes to vanilla costumes. 🙂 (I also swapped the order here as I think it's better to have the preferred/recommended type on the left, but that's just personal nitpick.)

Reply:
Done!